### PR TITLE
Fix some wrong coding reported by Thomas Munro.

### PR DESCRIPTION
**Pull request by thoshiai**
Some incorrect coding are fixed.
This advice is reported from pg-hackers by Thomas Munro.

### CODE REVIEW

File: src/backend/commands/createas.c [hunk 1]
```diff
@@ -388,7 +388,7 @@ ExecCreateTableAs(CreateTableAsStmt *stmt, const char *queryString,
 							&& strcmp(aggname, "count") != 0
 							&& strcmp(aggname, "avg") != 0
 						)
-							elog(ERROR, "Aggrege function %s is not supported", aggname);
+							elog(ERROR, "aggrege function %s is not supported", aggname);
```
Comment (yugo-n) [inline src/backend/commands/createas.c]: This should be fixed to "aggregate" not "aggrege", as pointed out by Thomas.

> s/Aggrege/aggregate/

This is same in regression test fixes.

File: src/bin/psql/describe.c [hunk 2]
```diff
@@ -1692,6 +1692,8 @@ describeOneTableDetails(const char *schemaname,
 	/* TODO: This will supported when sversion >= 130000 (or later). */
 	if (pset.sversion >= 120000)
 		tableinfo.isivm = strcmp(PQgetvalue(res, 0, 15), "t") == 0;
+	else
+		tableinfo.isivm = NULL;
```
Comment (yugo-n) [inline src/bin/psql/describe.c]: tableinfo.isivm = false; is correct rather than NULL because this is boolean.

**Comment by thoshiai** [comment]
Thank you for your review, Nagata-san. I fixed mistake.
Please check and merge it, if there is no problem.

**Comment by yugo-n** [comment]
Thanks. I confirmed it's ok.

**Pull request merged**